src/components/20-molecules/image-upload/index.js

```diff
@@ -0,0 +1,433 @@
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+/* eslint-disable import/no-extraneous-dependencies */
+import '@axa-ch/input-file';
+import { classMap } from 'lit-html/directives/class-map';
+
+import {
+  AddSvg,
+  DeleteForeverSvg,
+  ClearSvg,
+  AttachFileSvg,
+} from '@axa-ch/materials/icons';
+
+/* icon isolated from others, because it's a component specific icon */
+import { ImageUploadGroupSvg } from './icons';
+
+/* eslint-disable import/no-extraneous-dependencies */
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+import compressImage from './utils/imageCompressor';
+
+const AddIcon = svg([AddSvg]);
+const AttachFileIcon = svg([AttachFileSvg]);
+const DeleteForeverIcon = svg([DeleteForeverSvg]);
+const ClearIcon = svg([ClearSvg]);
+const ImageUploadGroupIcon = svg([ImageUploadGroupSvg]);
+
+const ACCEPTED_FILE_TYPES = 'image/jpg, image/jpeg, application/pdf, image/png';
+
+/* helperfunctions */
+export const getBytesFromKilobyte = kilobyte => 1024 * kilobyte;
+
+class AXAImageUpload extends LitElement {
+  static get tagName() {
+    return 'axa-image-upload';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      inputFileText: { type: String },
+      maxSizeOfSingleFileKB: { type: Number },
+      maxSizeOfAllFilesKB: { type: Number },
+      maxNumberOfFiles: { type: Number },
+      showImageOverview: { type: Boolean },
+      icon: { type: String },
+      deleteStatusText: { type: String },
+      addStatusText: { type: String },
+      fileTooBigStatusText: { type: String },
+      filesTooBigStatusText: { type: String },
+      tooManyFilesStatusText: { type: String },
+      orText: { type: String },
+      infoText: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.inputFileText = 'Upload file';
+    this.maxSizeOfSingleFileKB = 100;
+    this.maxSizeOfAllFilesKB = 500;
+    this.maxNumberOfFiles = 10;
```

Comment:
Please read row 12 on this page: https://confluence.axa.com/confluence/display/AEM/Upload+Control
I think it's useful to set it default to 10, because our backend system can handle max. 10 files.